src/System.Runtime.WindowsRuntime/src/System.Runtime.WindowsRuntime.csproj.disabled

```diff
@@ -130,8 +121,5 @@
     <Compile Include="System\Windows\TokenizerHelper.cs" />
     <Compile Include="System\Windows\UI\Color.cs" />
   </ItemGroup>
-  <ItemGroup>
-    <None Include="project.json" />
-  </ItemGroup>
```

Comment:
Lots of project still have these, if you feel like deleting the others. There's also src\System.Security.Cryptography.Xml\src\project.json which I guess shouldn't be there.